### ISSUE TITLE
Found some of the missing glottocodes

### ISSUE THREAD

**Issue by MacdonaldJoshuaCaleb** [D-PLACE/dplace-dataset-pulotu#3]
Hi,

double check of course, but I'm pretty sure I have found the following missing glottocodes for the dataset

Subanun - cent2089
Ifugao - tuwa1243
tagbanua - tagb1258
dusun - cent2100
kalinga - lubu1243

**Comment by MacdonaldJoshuaCaleb** [comment]
also 

futuna-west  west2515

**Comment by MacdonaldJoshuaCaleb** [comment]
think I found one more, and one possible correction to what I suggested previously

'futuna-west' should perhaps be futu1245 based on the language name in the cldf version of the data (which is Futuna-Aniwa) as glottolog differentiates these

https://glottolog.org/glottolog?search=futuna#6/-16.923/176.039

"Eastern-toraja" -> tora1262

That would just leave Berawan, which seems to lack sufficient information to nail down, though this could perhaps be done by going back to the original ethnography

**Comment by Joseph-Watts** [comment]
Thanks Joshua, we will look into this when we get the opportunity. I think the current lead editor of Pulotu (Oliver Sheehan) is on leave at the moment so there may be a bit of a delay.

**Comment by MacdonaldJoshuaCaleb** [comment]
> Thanks Joshua, we will look into this when we get the opportunity. I think the current lead editor of Pulotu (Oliver Sheehan) is on leave at the moment so there may be a bit of a delay.

Of course, thanks for maintaining this wonderful data-set and dp-lace in general.
